clang/test/CodeGen/conditional-operator.c

```diff
@@ -0,0 +1,36 @@
+// RUN: %clang_cc1 -disable-llvm-passes -emit-llvm %s -o - | FileCheck %s
+// RUN: %clang_cc1 -disable-llvm-passes -emit-llvm -menable-no-infs -fapprox-func\
+// RUN: -funsafe-math-optimizations -fno-signed-zeros -mreassociate -freciprocal-math\
+// RUN: -ffp-contract=fast -ffast-math %s -o - | FileCheck %s
```

Comment:
Redundantly using -ffast-math and all the individual flags?

Reply:
These flags correspond to how the clang driver `-ffast-math` are expanded to the individual flags passed to `-cc1`.

Several tests (e.g. `clang/test/Parser/fp-floatcontrol-syntax.cpp` and `clang/test/CodeGen/math-errno.c`) use similar command lines for testing fast-path behavior and I was trying to be consistent with them.